pages/team.js

```diff
@@ -0,0 +1,95 @@
+// eslint-disable-next-line no-restricted-imports
```

Comment:
Can you remove this line and then delete the next line?

You don't actually need to import React on any file in the pages folder. That's done for you.